### PR TITLE
Add PaymentSource.scannable_code

### PR DESCRIPTION
**Pull request by rezigned**
Add `PaymentSource.scannable_code` along with other objects e.g. `Barcode`, `Document`, etc.

### CODE REVIEW

File: Omise/Models/Models.cs [hunk 1]
```diff
@@ -489,6 +521,44 @@ public partial class Dispute : ModelBase {
         }
     }
 
+    public partial class Document : ModelBase {
+        [JsonProperty("download_uri")]
+        public string DownloadURI { get; set; }
+        [JsonProperty("filename")]
+        public string filename { get; set; }
```
Comment (bajang7mc) [inline Omise/Models/Models.cs]: Should it be capitalize the first letter?

File: Omise/Models/Models.cs [hunk 2]
```diff
@@ -489,6 +521,44 @@ public partial class Dispute : ModelBase {
...
+        [JsonProperty("location")]
+        public string location { get; set; }
```
Comment (bajang7mc) [inline Omise/Models/Models.cs]: Here too.

File: Omise/Models/Models.cs (hunk 1 above)
Comment (rezigned) [reply Omise/Models/Models.cs]: Good catch! Thank you.

**Review by bajang7mc — APPROVED** [review]
👍

**Pull request merged**